cli/waiter/subcommands/maintenance.py

```diff
@@ -57,13 +57,15 @@ def check_maintenance(clusters, args, enforce_cluster):
 def start_maintenance(clusters, args, enforce_cluster):
     """Sets the token in maintenance mode by updating the token user metadata fields"""
     token_name = args['token']
+    timeout_secs = args['timeout']
     kill_services_option = args.pop('kill_services', 'force_kill')
     cluster, existing_token_data, existing_token_etag = _get_existing_token_data(clusters, token_name, enforce_cluster)
     json_body = existing_token_data
     update_doc = {"maintenance": {"message": args['message']}}
     json_body.update(update_doc)
     return_code, token_etag = _update_token(cluster, token_name, existing_token_etag, json_body)
     if return_code == 0:
+        print_info(f'{token_name} maintenance mode activated on cluster {cluster}.')
```

Comment:
`token.domain.com maintenance mode activated` seems confusing. Maybe this is better?:
```suggestion
        print_info(f'maintenance mode activated for {terminal.bold(token_name)} on cluster {terminal.bold(cluster)}.')
```